src/mahoji/lib/abstracted_commands/puroPuroCommand.ts

```diff
@@ -42,10 +42,10 @@ export const puroOptions: PuroImpling[] = [
 		item: null
 	},
 	{
-		name: 'Dragon Implings',
-		hunterLevel: 83,
+		name: 'High-tier Implings',
+		hunterLevel: 58,
 		spell: true,
-		item: getOSItem('Dragon impling jar')
+		item: getOSItem('Nature impling jar')
```

Comment:
Not sure if anyone has a better solution for this

Reply:
Make it `item: null` like the one above?

Reply:
I can do that, just not sure how id handle the switch in the activity file then

Reply:
Made some changes although it could probably be condensed or tidied up better, think I'll need some assistance from yourself or another contributor